CommonTools/RecoAlgos/plugins/TOFPIDProducer.cc

```diff
@@ -0,0 +1,308 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+using namespace std;
+using namespace edm;
+
+
+
+
+
+class TOFPIDProducer : public edm::stream::EDProducer<> {  
+  static constexpr char t0Name[] = "t0";
```

Comment:
code organization: private members should go in the explicitly-labeled "private" section